### DISCUSSION TITLE
iPhone as a webcam is supported on OpenVINO Notebooks

### DISCUSSION THREAD

**Discussion by raymondlo84** [openvinotoolkit/openvino_notebooks#672] [Show and tell]
![image](https://user-images.githubusercontent.com/1720147/202892492-a5dafb73-81db-46e2-aa68-a049da7a71a6.png)

You can use your iPhone as the webcam in our notebook examples. You all you have to do is to change the index to 1 (or 2 ...depends on your setup).

Cheers.

**Comment by brmarkus** [comment]
Can you share more about your setup, please?
Latest MacBook(Pro), latest iPhone, using which MacOS and which iOS? Or using MS-Windows? How is the iPhone connected to your machine, wireless or via USB-cable?
Were you required to install any driver/package, have you created a Python-virtual-camera, a v4lv2-loopback?

I just tried it, using my iPhone 8 with iOS 15.7.1 under Ubuntu 20.04 LTS - nope, not recognized as video-device.

**Reply by raymondlo84** [reply]
I'm using Mac 13.0.1 as it supports that natively (wired and wirelessly). And another way is to use OBS to enable the virtual webcam and have the iPhone as a webcam source (wired).

**Reply by raymondlo84** [reply]
https://obs.camera/docs/getting-started/ios-camera-plugin-usb/ :)
